### ISSUE TITLE
Request: Use currency symbol instead of Convert icon when converting currency

### ISSUE THREAD

**Issue by filliph** [deanishe/alfred-convert#67]
When converting currency into multiple units at once, it would make it easier to differentiate if a currency symbol was used instead of the Convert icon:

<img width="572" alt="image" src="https://user-images.githubusercontent.com/9424286/84894012-15792580-b098-11ea-9b99-0237cdcaf41c.png">

Thanks for considering :)

**Comment by deanishe** [comment]
I'm not sure I follow you. Do you mean the appropriate icon for each currency, or just one currency icon that's different from the normal icon?

**Comment by filliph** [comment]
The appropriate icon for each currency. Here's a badly made mockup:
<img width="572" alt="84894012-15792580-b098-11ea-9b99-0237cdcaf41c" src="https://user-images.githubusercontent.com/9424286/84908211-174ce400-b0ac-11ea-9335-0603d4492db5.png">
(But also using the appropriate icons for the other currencies 😅)

**Comment by deanishe** [comment]
Right, that makes more sense than the other way. Do you know of a ready-made collection of currency icons? The workflow supports nearly 200 fiat currencies, and several times as many cryptocurrencies.

There is **no way** I'm going to try and make even a fraction of those myself. I suppose flag icons would be an alternative, but I still don't fancy mapping hundreds of currencies to icons.

**Comment by filliph** [comment]
I've found these for crypto: https://github.com/allienworks/cryptocoins | https://github.com/spothq/cryptocurrency-icons
And these other MIT-licensed Alfred workflows that have flags ready-mapped: https://github.com/tomy0000000/coon | https://github.com/plummm/alfred3-workflow-CurrencyX

I can't seem to find currency symbols for fiat, but it might be more recognisable to use flags for that anyway. 

If neither of those crypto projects keep their icons in the same format as you'd need them, I might be able to help convert them if you give me a format / size to aim for 😄

**Comment by deanishe** [comment]
Good finds. None of them are really in the right format/size, but that's easy enough to fix with a script. Workflow icons should be 128px (they're 64px in size, and double it for Retina screens).

I still can't believe that fella called his workflow "Coon". Someone opened an issue [explaining that the name is inapproriate](https://github.com/tomy0000000/Coon/issues/2), but I don't think he got across exactly how inappropriate it is.

I'll have a think about it. The workflow itself has no idea what kind of units it's showing, so that would need to be handled, too.

Perhaps I could add something generic, like "look for an icon named `<unit>.png`", and then people can add their own icons for other units, too, if they want. Also no need to add all the icons at once, I suppose.

**Comment by filliph** [comment]
> I still can't believe that fella called his workflow "Coon". Someone opened an issue [explaining that the name is inapproriate](https://github.com/tomy0000000/Coon/issues/2), but I don't think he got across exactly how inappropriate it is.

Yeah... That's one of the reasons I went looking for alternatives and found this project. Made sure to delete it off my computer as well.

> I'll have a think about it. The workflow itself has no idea what kind of units it's showing, so that would need to be handled, too.
> 
> Perhaps I could add something generic, like "look for an icon named `<unit>.png`", and then people can add their own icons for other units, too, if they want. Also no need to add all the icons at once, I suppose.

Yeah, if there was a set of icons built-in, as well as a sub-folder in `Workflow Data/net.deanishe.alfred-convert` where we could drop custom icons, that'd probably be the optimal solution 😄 

If I knew Python I'd dig around to see if I could help with a PR, my forte is PHP (pls no bully 😭)

**Comment by deanishe** [comment]
> Yeah, if there was a set of icons built-in, as well as a sub-folder in Workflow Data/net.deanishe.alfred-convert where we could drop custom icons, that'd probably be the optimal solution 😄

Exactly what I was thinking.

> Yeah... That's one of the reasons I went looking for alternatives and found this project. Made sure to delete it off my computer as well.

I'm going to create another issue to try to make clear that "culturally inappropriate" is a massive understatement. The author is Taiwanese, and I think he just didn't get how bad the word is.
